source/process/mobile-release-process.md

```diff
@@ -9,58 +9,137 @@ Notes:
 - All cut-off dates are based on 10am (San Francisco Time) on the day stated.
 - T-minus counts are measured in "working days" (weekdays other than major holidays in the US and Canada) prior to release day.
 
-### A. (T-minus 8 working days) Code Complete and Release Candidate Cut 
+### A. (T-minus 12 working days) Cut-off for merging major features
 
-1. PM:
-    - Submit pull request for changelog
+No pull requests for major features should be **merged** to the current release after this date. In special cases, exceptions can be made by the Release Manager.
+
+1. Release Manager:
+    - Post this checklist in Release Checklist channel
+    - Draft Changelog in a WIP PR with updates for highlights, feature additions and bug fixes
+    - Start posting a daily Zero Bug Balance query (posted until zero bugs or day of release)
```

Comment:
This will be tricky because we dont have a way of differenciating RN and platform bugs. Maybe we need a label or new JIRA field?

Reply:
Bug agree we should be doing this. 

We should specify this is in the Native Mobile Apps channel

Reply:
@esethna Would this link work: https://mattermost.atlassian.net/projects/MM/versions/12427/tab/release-report-todo?